pkgs/development/python-modules/karton-yaramatcher/default.nix

```diff
@@ -24,7 +24,6 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace requirements.txt \
-      --replace "karton-core==4.0.5" "karton-core" \
       --replace "yara-python==4.0.2" "yara-python" \
```

Comment:
This looks that the change for `yara-python` is [out-dated](https://github.com/CERT-Polska/karton-yaramatcher/blob/master/requirements.txt#L2) and can be removed, too.